plugins/dominant-color-images/helper.php

```diff
@@ -58,6 +58,13 @@ function dominant_color_get_dominant_color_data( $attachment_id ) {
 		$file = get_attached_file( $attachment_id );
 	}
 	add_filter( 'wp_image_editors', 'dominant_color_set_image_editors' );
+
+	/**
```

Comment:
```suggestion
	/*
```
Seems it's local variable so do we needs to use multiline comment per https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#5-inline-comments

Reply:
I don't think that's right. Without `/**` then the phpdoc is not parsed.

Reply:
The docs you link to are talking about a regular multiline comment, not one that serves as a doc block.